clar2wasm/src/words/sequences.rs

```diff
@@ -1181,7 +1181,7 @@ impl ComplexWord for ReplaceAt {
             builder.local_tee(repl_len).unop(UnaryOp::I32Eqz).if_else(
                 None,
                 |then| {
-                    then.i32_const(ErrorMap::Panic as i32)
+                    then.i32_const(ErrorMap::BadTypeConstruction as i32)
```

Comment:
Is this a `BadTypeConstruction`? Looks more like an incorrect value usage.

Reply:
Damn, I clicked «request changes» instead of «comment». My bad.

Reply:
Yeah! More related to values.

Reply:
Since this error happens only here, would it make sense to give it a more explicit name? `IncorrectValue` for example?

Reply:
Well, okay, if it's an [artifact from the interpreter](https://github.com/stacks-network/stacks-core/blob/6f498f80a1e6495aaf7789d3ff6c987663aaf011/clarity/src/vm/types/mod.rs#L314-L315), there isn't much we can do...